docs/reference/commandline/tye.md

```diff
@@ -29,6 +29,7 @@ For example, [`tye run`](tye-run.md) runs an application. Each command defines i
 
 | Command                                       | Function                                                            |
 | --------------------------------------------- | ------------------------------------------------------------------- |
-| [tye init](tye-init.md) | Scaffolds a tye.yaml file representing the application.                          |
-| [tye run](tye-run.md)               | Runs an application.                                     |
-| [tye deploy](tye-deploy.md)               | Deploys an application.                                             |
+| [tye init](tye-init.md)                       | Scaffolds a tye.yaml file representing the application.             |
+| [tye run](tye-run.md)                         | Runs an application.                                                |
+| [tye deploy](tye-deploy.md)                   | Deploys an application.                                             |
+| [tye cleanup](tye-cleanup.md)                 | Deletes a deployed application.                                     |
```

Comment:
clean?

Reply:
I don't really love either of these tbh. I called it cleanup expecting a debate on the PR. I don't like `tye clean` because we have tye build, and it makes it seem like `tye build` and `tye clean` are related.

Some ideas:

- `tye cleanup` (current)
- `tye clean` (I this it will confuse people)
- `tye teardown` 
- `tye undeploy`
- `tye delete`

Reply:
tye undeploy ftw

Reply:
I think `tye clean` or `tye delete` makes the most sense, and it depends on what we want this command to do.

If we just want it to delete any resources that are in the cluster, I think we should go with just `delete` and maybe allow subcomponents (like accepting a name for a service).

If we want this command to reset or modify state of the cluster besides deletion, then `clean` may be more appropriate.

Reply:
> If we just want it to delete any resources that are in the cluster, I think we should go with just

It definitely refers to the cluster. I'm kinda leaning towards `undeploy` because:

- it's clear that it undoes deploy
- it's a name you wouldn't guess, so you don't end up accidentally nuking the world

Reply:
Agreed with `undeploy` offline

Reply:
Ok. going with undeploy.